resource-synchronizer/poet_ready.py

```diff
@@ -32,12 +32,12 @@ def rollback(code_dir, rollback_dir):
             dest_path_file = path_file.replace(rollback_dir, code_dir)
             shutil.copy2(path_file, dest_path_file)
 
-    cleanup()
+    cleanup(rollback_dir)
 
 
 def main():
     code_dir = Path(sys.argv[0]).resolve(strict=True)
```

Comment:
Should this be updated to `argv[1]` as well?

Reply:
aiy yes, thank you!